boa/src/builtins/object/mod.rs

```diff
@@ -410,6 +410,36 @@ pub fn make_object(_: &Value, args: &[Value], ctx: &mut Interpreter) -> ResultVa
     Ok(object)
 }
 
+/// `Object.create( proto, [propertiesObject] )`
+///
+/// Creates a new object from the provided prototype.
+///
+/// More information:
+///  - [ECMAScript reference][spec]
+///  - [MDN documentation][mdn]
+///
+/// [spec]: https://tc39.es/ecma262/#sec-object.create
+/// [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/create
+pub fn create_builtin(_: &Value, args: &[Value], interpreter: &mut Interpreter) -> ResultValue {
```

Comment:
we should rebase on the latest commit on master, `init` and `create` has been merged into just init which would allow us to call this `create` instead of `create_builtin`